pages/books/Search.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+import { styled, createTheme, ThemeProvider } from '@mui/material/styles';
+import InputBase from '@mui/material/InputBase';
+import SearchIcon from '@mui/icons-material/Search';
+
+
+
+
+export default function Search() {
```

Comment:
see comment here - https://github.com/sprathap1096/CS554_Final_Project_Emanon/pull/5#discussion_r867399376

Reply:
do u want me to move this to components as well?

Reply:
done